tests/unit_test/data_processor/data_processor_test.py

```
@@ -1667,6 +1669,510 @@
+            user="user",
+            schedule_time=CustomActionDynamicParameters(parameter_type=ActionParameterType.slot.value,
+                                                        value="location"),
+            schedule_action="callback_script2",
+            timezone="Asia/Kolkata",
+            response_text="Action schedule",
+            params_list=[CustomActionRequestParameters(key="name", parameter_type="slot", value="name", encrypt=True),
+                         CustomActionRequestParameters(key="user", parameter_type="value", value="1011", encrypt=False)]
+        ).save()
+
+    def test_get_slot_actions(self, save_actions):
+        processor = MongoProcessor()
```

Ruff

Redefinition of unused `processor` from line 14

(F811)